build/env.sh

```diff
@@ -26,5 +26,15 @@ export GOPATH
 cd "$ethdir/go-ethereum"
 PWD="$ethdir/go-ethereum"
 
+# Prebuild SVM and set up CGO_LDFLAGS if the build is svm-enabled
+if [ "$SVM" = "true" ]; then
+    # Check that we're not using xgo
+    if [[ "$*" = *"xgo"* ]]; then
+        echo "Cross-builds are not yet supported with EVM-RS enabled"
+        exit 1
+    fi
+    . build/build_evm-rs.sh
```

Comment:
Use `source` instead of  `.`, for POSIX sh compatibility.

Reply:
Wait a second...

bb2d121 

![Screenshot_20190521_075500](https://user-images.githubusercontent.com/45600330/58097776-c4011d00-7b9d-11e9-8efd-d3cb26a22661.png)

Reply:
I read this error message the wrong way ;)